### PR TITLE
vw_golf_mk4: a bit of progress

### PR DESCRIPTION
**Pull request by rusefillc**
A bit more of the same - English translation as comments

### CODE REVIEW

**Comment by rusefillc** [comment]
@adeebshihadeh @sshane poke :)

**Comment by sshane** [comment]
@jyoung8607 does this look good?

File: vw_golf_mk4.dbc [hunk 1]
```diff
@@ -303,7 +303,8 @@ BO_ 1408 Motor_Flexia: 8 XXX
  SG_ Multiplex_Schalter_Motor_Flexia M : 0|1@1+ (1,0) [0|0] "" XXX
 
 BO_ 1416 Motor_7: 8 XXX
- SG_ Frei_Motor_7_3 : 40|24@1+ (1,0) [0|0] "" XXX
+ SG_ Oltemperatur : 56|8@1+ (1,0) [0|0] "" XXX
+ SG_ Frei_Motor_7_3 : 40|16@1+ (1,0) [0|0] "" XXX
```
Comment (jyoung8607) [inline vw_golf_mk4.dbc:307]: Let's revert this change. If we're going to clean up sorting of signals, they would go in ascending order rather than descending, and it should be done in a standalone PR for that purpose rather than intermixed with translation/commentary updates. And I'm not in a great hurry to perform/review that sort at this particular time.

File: vw_golf_mk4.dbc [hunk 2]
```diff
@@ -321,7 +322,8 @@ BO_ 1160 Motor_6: 8 XXX
  SG_ Frei_Motor_6_4 : 58|2@1+ (1,0) [0|0] "" XXX
  SG_ ltemperaturschutz : 57|1@1+ (1,0) [0|0] "" XXX
  SG_ GRA_Bremseingriff_Freigabe : 56|1@1+ (1,0) [0|0] "" XXX
- SG_ Frei_Motor_6_3 : 40|16@1+ (1,0) [0|0] "" XXX
+ SG_ Frei_Motor_6_3 : 48|8@1+ (1,0) [0|0] "" XXX
+ SG_ Ruckmeldung_Momenten : 40|8@1+ (0.39,0) [0|100] "" XXX
```
Comment (jyoung8607) [inline vw_golf_mk4.dbc:326]: Same, revert.

**Review by jyoung8607 — CHANGES_REQUESTED** [review]

File: vw_golf_mk4.dbc (hunk 1 above)
Comment (rusefillc) [reply vw_golf_mk4.dbc:307]: @jyoung8607 let's confirm it's not a misunderstanding. I belive this is NOT a sorting change here.
"Frei_Motor_XX_XX" in both cases seems like "unused unmapped areas" and in both cases I am reducing the lenght of "unused unmapped Frei_Motor" while adding a new field definition which takes a part of what was "unmapped"

like here I believe that I am ading Oltemperatur and reduce unmapped placeholder from 24 bits to 16 bits

Question: shall I revent addition of the new field?

File: vw_golf_mk4.dbc (hunk 2 above)
Comment (rusefillc) [reply vw_golf_mk4.dbc:326]: Please confirm if I should revert new field "Ruckmeldung_Momenten"

**Comment by rusefillc** [comment]
@jyoung8607 did you have a chance to revisit?
@sshane @adeebshihadeh poke :)

**Comment by rusefillc** [comment]
@jyoung8607 @sshane @adeebshihadeh poke

**Comment by sshane** [comment]
@jyoung8607 gtm?

**Review by jyoung8607 — APPROVED** [review]
Eventually the filler/placeholder signals tweaked in this PR (such as `Frei_Motor_7_3`) should really be discarded, and we should replace the whole of Motor_7 (among others) with its current/modern PQ35/PQ46 equivalent as we've done with other messages, but this is close enough for now.

**Comment by rusefillc** [comment]
@sshane poke?
